lib/fake_braspag/sales.rb

```diff
@@ -40,6 +40,16 @@ class Sales < Sinatra::Base
       end
     end
 
+    put '/:PaymentId/conciliate' do
+      order = Order.find(params[:PaymentId])
+      if order.present? && order.PaymentMethod == 'Boleto' && ResponseToggler.enabled?('conciliate')
```

Comment:
I don't know why, but, this way I got this errror:
Undefined method ```PaymentMethod``` for nil class... :disappointed: